models/netsuite2/intermediate/base/int_netsuite2__accounts.sql

```diff
@@ -12,17 +12,45 @@ account_types as (
     from {{ var('netsuite2_account_types') }}
 ),
 
+account_hierarchy as (
```

Comment:
Fivetran: We'll also want to consider if we want to limit this to instances where the customer is using subsidiaries.

Reply:
Seemed to work.